File: 2.0/java/orders/getOrder.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.NumberOrder;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        Long orderId = 379506003L;
        NumberOrder order = callfireClient.ordersApi().getOrder(orderId, "status,totalCost");
    }
}

File: 2.0/csharp/orders/getOrder.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        const int orderId = 379506003;
        NumberOrder order = client.OrdersApi.GetOrder(orderId, "status,totalCost");
    }
}
